packages/@expo/cli/e2e/__tests__/export/static-splitting.test.ts

```diff
@@ -121,7 +121,7 @@ describe('exports static with bundle splitting', () => {
     // "_expo/static/js/web/links-4545c832242c66b83e4bd38b67066808.js.map",
     // "_expo/static/js/web/styled-93437b3b1dcaa498dabb3a1de3aae7ac.js.map",
     expect(mapFiles).toEqual(
-      ['\\[post\\]', '_layout', 'about', 'asset', 'index', 'index', 'links', 'styled'].map((file) =>
+      ['_layout', '_post_', 'about', 'asset', 'index', 'index', 'links', 'styled'].map((file) =>
         expect.stringMatching(new RegExp(`_expo\\/static\\/js\\/web\\/${file}-.*\\.js\\.map`))
```

Comment:
We should have a test to account for cases where a user already has `_post_` in the project.